project_description/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Project Description
+===================
+
+This module extends the functionality of projects to support setting a
+description for each project and allow you to search by that description.
+
+Usage
+=====
+
+To set a description for a project, you need to:
+
+#. Go to *Project > Project > Projects*.
+#. Edit or create one.
+#. You will see a new *Description* tab, with an HTML area to set the project's
+  description.
+
+.. figure:: static/description/image.png
+   :alt: Project form with the description tab
+
+To search for a project by its description, you need to:
+
+#. Go to *Project > Project > Projects*.
+#. Enter any word in the search box.
+#. Select *Search Description*.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/11.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/project/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
```

Comment:
Update the text according to [latest template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)